docker-selenium.yaml

```diff
@@ -134,16 +132,22 @@ subpackages:
         - name: "Check selenium base status"
           uses: test/daemon-check-output
           with:
-            start: "/opt/bin/entry_point.sh"
+            setup: |
+              adduser -D -H seluser
+              chown seluser:seluser /tmp /var/tmp
+              mkdir -p /var/run/supervisor
+              chown seluser:seluser /var/run/supervisor
+            start: |
+              su seluser -c /opt/bin/entry_point.sh
             timeout: 60
             expected_output: |
               supervisord started
 
-  - name: ${{package.name}}-config
+  - name: ${{package.name}}-compat
```

Comment:
We seem to be changing a subpackage name? Have you verified it is not being used anywhere else in images?

Reply:
It is not used anywhere else